### PR TITLE
Updated default fossil fuel emissions factors

### PR DESCRIPTION
**Pull request by shorowit**
## Pull Request Description

Updated default fossil fuel emissions factors to include pre-combustion (e.g., methane leakage for natural gas), per RESNET. Values are derived from ASHRAE Standard 189.1 Addendum m, to include pre-combustion, but w/ a 100-year GWP time horizon.

Fuel Type | CO2e Old (lb/MBtu) | CO2e New (lb/MBtu)
-- | -- | --
Natural Gas | 117.6 | 147.3
Fuel Oil | 161 | 195.9
Propane | 136.6 | 177.8

Also renamed the emissions type (that the default values apply to) from "CO2" to "CO2e".

## Checklist

Not all may apply:

- [x] ~EPvalidator.xml has been updated~
- [x] Tests (and test files) have been updated
- [x] Documentation has been updated
- [x] Changelog has been updated
- [x] `openstudio tasks.rb update_measures` has been run
- [x] No unexpected regression test changes on CI (checked comparison artifacts)

### CODE REVIEW

File: BuildResidentialHPXML/tests/build_residential_hpxml_test.rb [hunk 1]
```diff
@@ -841,7 +841,7 @@ def _set_measure_argument_values(hpxml_file, args)
       args['site_time_zone_utc_offset'] = '-6'
     elsif ['extra-emissions-fossil-fuel-factors.xml'].include? hpxml_file
       args['emissions_scenario_names'] = 'Scenario1, Scenario2'
-      args['emissions_types'] = 'CO2, SO2'
+      args['emissions_types'] = 'CO2e, SO2'
```
[the remark this answers is not in the document]
Comment (shorowit) [reply BuildResidentialHPXML/tests/build_residential_hpxml_test.rb:844]: @ejhw Hmm, are we talking about two different things? I'm referring to upstream methane leakage during extraction of natural gas, which I believe is the primary "pre-combustion" effect for fossil fuels, not methane leakage on site.

[the remark this answers is not in the document]
Comment (ejhw) [reply BuildResidentialHPXML/tests/build_residential_hpxml_test.rb:844]: Right, CO2e should include both. I didn't think those values from EIA included either upstream or BTM leakage. Do they include upstream?

File: docs/source/workflow_inputs.rst [hunk 2]
```diff
@@ -266,20 +266,20 @@ For each scenario, fuel emissions factors can be optionally entered as an ``/HPX
...
-  wood          --             --             --
-  wood pellets  --             --             --
-  ============  =============  =============  =============
+  ============  ==============  =============  =============
+  Fuel Type     CO2e [lb/MBtu]  NOx [lb/MBtu]  SO2 [lb/MBtu]
+  ============  ==============  =============  =============
+  natural gas   147.3           0.0922         0.0006
+  propane       177.8           0.1421         0.0002
+  fuel oil      195.9           0.1300         0.0015
+  coal          --              --             --
```
Comment (joseph-robertson) [inline docs/source/workflow_inputs.rst:279]: The coal default value intentionally goes away?

Comment (shorowit) [reply docs/source/workflow_inputs.rst:279]: Yes, it was pulled from a different data source and I don't think it's apples-to-apples with the other fossil fuels. I probably shouldn't have included it to begin with.

**Pull request merged**